style.css

```diff
@@ -334,3 +334,77 @@ button {
   line-height: 20px;
   color: #253858;
 }
+
+.contact-form {
+  background-image: url(src/Contact-me.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contactContainer {
+  background-image: url(src/contact-shapes-mobile.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+}
+
+.form-title {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
+}
```

Comment:
- [ ] Kindly add some `padding-top`, to be like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155250198-9516765f-0e5a-4f89-8bce-8820a7b31d73.png) | ![image](https://user-images.githubusercontent.com/35535130/155250213-56b062c3-7425-4efd-ab55-f6003476ed1b.png) |